### PR TITLE
feat: add profile to launch harbor as external registry

### PR DESCRIPTION
**Pull request by guilhem-barthes**
## Description

Describe how to start a cluster with Harbor

Fixes  FL-1477 & FL-1479

## Companion PR

- https://github.com/Substra/substra-backend/pull/862

### CODE REVIEW

File: docs/source/how-to/developing-substra/harbor.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+=====================
+
+First-time set-up (specific to in-cluster)
+------------------------------------------
+
+As ``sudo``, add ``127.0.0.1       harbor.harbor.svc.cluster.local`` to ``/etc/hosts``
+
+      .. code:: bash
+
+         echo "127.0.0.1       registry.org-2.com" | sudo tee -a /etc/hosts
```
Comment (SdgJlbl) [inline docs/source/how-to/developing-substra/harbor.rst:21]: 👀 
I think that the command does not do what the text says.

Comment (guilhem-barthes) [reply docs/source/how-to/developing-substra/harbor.rst:21]: Thanks, the second part is the correct one

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Pull request merged**